android/README.md

```diff
@@ -228,14 +228,6 @@ Helper method which should be called from the activity's `onResume` method.
 
 This is a static method.
 
-#### onNewIntent(intent)
```

Comment:
* The commit '[iOS] Add initial CallKit support' is in master as https://github.com/jitsi/jitsi-meet/commit/8d11b3024ec5de66dbd84798af001402bcb58844.

* The commit '[RN] Simplify handling of universal / deeps links' is not merged because it (1) breaks the source code compatibility of the SDK, (2) takes utility functionality away from SDK clients/consumers, and (3) is really not impeding the development of the CallKit support.